packages/server/src/arpa_reporter/services/validate-upload.js

```diff
@@ -249,27 +283,29 @@ async function validateSubrecipientRecord({
     || existing.updated_at
     ) isOwnedByThisUpload = false;
 
-    // the record has already been validated before this method was invoked. how
-    // did the validation go?
-    const isRecordValid = recordErrors.length === 0;
+    // the record has already been validated before this method was invoked, or we found an error above. how did the validation go?
+    const isRecordValid = recordErrors.length === 0 && errors.length === 0;
```

Comment:
Was this an existing bug? In the past we never checked the length of `errors` to determine if `isRecordValid` so curious if you noticed this was incorrect and changed it.

Reply:
This is supposed to reflect something Truman called out, which is that if we have a validation error we don't even want to create  or update the record in the DB. I can / should actually call this out more explicitly